docs/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.0.x-to-4.1.8.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Migrate from 4.1.7 to 4.1.8 - Strapi Developer Docs
+description: Learn how you can migrate your Strapi application from 4.1.7 to 4.1.8.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.0.x-to-4.1.8.html
+---
+
+# v4.0.x to v4.1.8 migration guide
+
+The Strapi v4.0.x to v4.1.8 migration guide upgrades versions of v4.0.6 through v4.1.7 to v4.1.8. The minimum configuration for `config/admin` now includes the API token `API_TOKEN_SALT`. Additionally, Strapi no long populates default values for the admin JSON web token in `config/admin`. Initial values are generated and stored in the .env file during project creation. The migration to v4.1.8 consists of 2 steps:
```

Comment:
You can just use `admin JWT` I think, no need to spell it out entirely.

I think it would also be better to say "Strapi no longer generates random values for secrets in environments other than development"

Reply:
Can you throw something on my calendar for Thursday and we can step through this one together? (I have free time tmrw, Wednesday too but not sure if you wanted to stay that late)